spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require './spec/rails_helper'
+
+describe PostsController, type: :request do
+  describe 'test for index method' do
+    before(:each) do
+      get '/users/:id/posts'
+    end
+
+    it 'should test for request status index method' do
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214323927-7463c824-fb4c-4f95-9b3a-f33b3b3754e8.png)

**[Optional]**

- [ ] A test from the post model is not passing. Kindly fix it because tests only fail when there is an issue in your application behavior :+1: